plugins/metric/BetweennessCentrality.cpp

```diff
@@ -104,35 +129,11 @@ class BetweennessCentrality : public DoubleAlgorithm {
       stack<node> S;
       TLP_HASH_MAP<node, list<node>> P;
       MutableContainer<int> sigma;
```

Comment:
**sigma** variable is no longer used at this level, it should be an internal variable of compute...() functions.
But as it it not really used in computeDikjstra() it does not have to be defined (cf my proposal about your new version of the Dikjstra contructor).

Reply:
[idem](https://github.com/Tulip-Dev/tulip/pull/103#discussion_r279696990)

Reply:
[idem](https://github.com/Tulip-Dev/tulip/pull/103#discussion_r279754375)